pkgs/development/python-modules/amshan/default.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  construct,
+  paho-mqtt,
+  pyserial-asyncio,
+}:
+
+buildPythonPackage rec {
+  pname = "amshan";
+  version = "2.1.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "toreamun";
+    repo = "amshan";
+    rev = version;
+    hash = "sha256-aw0wTqb2s84STVUN55h6L926pXwaMSppBCfXZVb87w0=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    construct
+    paho-mqtt
+    pyserial-asyncio
+  ];
```

```suggestion
  build-system = [
    setuptools
  ];

  dependencies = [
    construct
    paho-mqtt
    pyserial-asyncio
  ];
```

New lingo, wheel is generally not required,